modules/install-and-configure-oadp-kubevirt.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="install-and-configure-oadp-kubevirt"]
+= Installing and configuring {oadp-short} with {VirtProductName}
+
+As a cluster administrator, you install {oadp-short} by installing the {oadp-short} Operator.
+
+The Operator installs link:https://velero.io/docs/v{velero-version}[Velero {velero-version}].
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Install the xref:../../../backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc#about-installing-oadp[OADP Operator] according to the instructions for your storage provider.
+
+. Install the xref:oadp-installing-dpa-1-3_installing-oadp-kubevirt[Data Protection Application (DPA)] with the `kubevirt` and `openshift` xref:../../../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[plugins].
+
+. Back up virtual machines by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[`Backup` custom resource (CR)].
```

Comment:
You can't have xrefs in a module, which is why your build is broken. In my comment yesterday, I suggested that you consider pulling these links into an `==Additional resources` heading instead of categorizing them as a `.Procedure`, which they aren't. This entire "procedure" is more like an overview of procedures to come.